pkg/apis/core/types_shoot.go

```diff
@@ -325,8 +325,31 @@ type ClusterAutoscaler struct {
 	ScaleDownUtilizationThreshold *float64
 	// ScanInterval how often cluster is reevaluated for scale up or down (default: 10 secs).
 	ScanInterval *metav1.Duration
+	// Expander defines the algorithm to use during scale up (default: least-waste).
+	// See: https://github.com/gardener/autoscaler/blob/machine-controller-manager-provider/cluster-autoscaler/FAQ.md#what-are-expanders.
+	Expander *string
```

Comment:
You can use something like 
```golang
type ExpanderMode string
const (
	ClusterAutoscalerExpanderLeastWaste ExpanderMode = "least-waste"
	ClusterAutoscalerExpanderMostPods ExpanderMode = "most-pods"
...

	Expander *ExpanderMode
...	
```

Reply:
Done [here](https://github.com/gardener/gardener/pull/4508/commits/ed626583966d98e816e697c1b4780373f0a3739d#diff-2a20577716c98d94c2d517871253ed0e37c0051d9caf6bc099f1edd1429c0ab8R335-R337).